src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ContactViewModels.cs

```diff
@@ -1,34 +1,34 @@
-using System;
+#nullable enable
 using System.ComponentModel.DataAnnotations;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
 {
     public class ContactViewModel
     {
-        public Guid Id { get; set; }
+        public string TeamName { get; set; } = string.Empty;
 
-        public string TeamName { get; set; }
+        public string TeamEmail { get; set; } = string.Empty;
 
-        public string TeamEmail { get; set; }
+        public string ContactName { get; set; } = string.Empty;
 
-        public string ContactName { get; set; }
+        public string ContactTelNo { get; set; } = string.Empty;
 
-        public string ContactTelNo { get; set; }
+        public ContactPermissions? Permissions { get; set; }
+
+        public record ContactPermissions
+        {
+            public bool CanUpdatePublication { get; set; }
```

Comment:
Why do we have this on a contact? Shouldn't we have a permission on publication permissions called `CanUpdateContact`?

Reply:
I've just changed the name to `CanUpdateContact` for now - it still uses `userService.CheckCanUpdatePublication`. The same work will also need to be done with external methodology, legacy releases, etc., so I intend to sort out all the permission changes in my current branch. I've left a comment in `PermissionsUtils` to remind me to do this for contacts.

Reply:
Would suggest just doing it as part of this PR. Like I'm not sure what time we're saving here by not doing it here?

Reply:
Discussed this with Nick on Slack - I'll make the changes on this PR.